biochemist-python/chapters/Non-Linear Regression Part 2.ipynb

```diff
@@ -0,0 +1,794 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Create your independent and dependent variables for curve fitting.\n",
+    "ydata = rates_df['initial velocities']\n",
+    "xdata = rates_df['pNPP (mM)']\n",
+    "\n",
+    "# Inspect your data to make sure it looks like a Michaelis-Menten plot\n",
+    "plt.plot(xdata, ydata, 'o', label='data')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0     95.238095\n",
+       "1     90.909091\n",
+       "2     87.500000\n",
+       "3     80.000000\n",
+       "4     66.666667\n",
+       "5     50.000000\n",
+       "6     41.176471\n",
+       "7     28.571429\n",
+       "8     16.666667\n",
+       "9      9.090909\n",
+       "10     4.761905\n",
+       "Name: pNPP (mM), dtype: float64"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create a function for the equation you want to fit. The function should accept \n",
+    "# as inputs the independent variable(s) and all the parameters to be fit.\n",
+    "def MM(SubConc, Vmax, Km):\n",
+    "    Vi = Vmax * SubConc/(Km + SubConc)\n",
+    "    return Vi\n",
+    "\n",
+    "# Test the function to see if it produces the expected output\n",
+    "MM(rates_df['pNPP (mM)'], 100, 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Common errors \n",
```

Comment:
I think this should go in its own lesson/section. Doesn't seem to fit here for me

Reply:
Do you think it would work earlier in the book? Maybe even in the introduction? Or maybe as an optional section of the introduction.